dart/lib/src/http_client/sentry_http_client.dart

```diff
@@ -83,7 +83,6 @@ class SentryHttpClient extends BaseClient {
     MaxRequestBodySize maxRequestBodySize = MaxRequestBodySize.never,
     List<SentryStatusCode> failedRequestStatusCodes = const [],
     bool captureFailedRequests = false,
-    bool sendDefaultPii = false,
     bool networkTracing = false,
```

Comment:
`networkTracing` can be replaced by `SentryOptions#isTracingEnabled`

Reply:
`SentryDioExtension` should be done the same

Reply:
I havevmade this change but in the tests, network tracing is being set true  sometimes and sometimes false. How do I set it if I remove this paraeeter?

Reply:
You've to adapt your tests to create the SentryOptions either with tracing enabled or disabled depending on your tests.